pkgs/development/python-modules/jupyterlab_server/default.nix

```diff
@@ -25,7 +25,11 @@ buildPythonPackage rec {
   };
 
   postPatch = ''
-    sed -i "/^addopts/d" pyproject.toml
+    substituteInPlace pyproject.toml \
+      --replace "--cov jupyterlab_server --cov-report term-missing --cov-report term:skip-covered" ""
+
+    # translation tests try to install additional packages into read only paths
```

Comment:
arch disables `test_translation_api` but i assumed it was just because it used internet https://github.com/archlinux/svntogit-community/commit/c764aeeefb3afaac6a7a37a69e6890b80656e2d4

Reply:
We can also go that route but I am not sure if that is enough to prevent it from installing the language packs.

See https://github.com/jupyterlab/jupyterlab_server/blob/v2.12.0/tests/test_translation_api.py#L27-L28